crowbar_framework/lib/crowbar/migrate.rb

```diff
@@ -31,6 +31,14 @@ def migrate!
           )
         end
       end
+
+      def migrate_to(level)
+        puts "Migrating database schema to #{level}..."
+        ActiveRecord::Migrator.migrate(
+          ActiveRecord::Migrator.migrations_paths,
+          level.to_i
```

Comment:
just to mention, that also seems to work for down migrations. so when a Cloud6 backup is older than the `current_version` it will migrate down